### PR TITLE
more changes to readme

### PR DESCRIPTION
**Pull request by sindhupalakodety**
Add relevant changes to readme. Please approve.

### CODE REVIEW

File: ReadMe.md [hunk 1]
```diff
@@ -2,13 +2,21 @@
...
+
+## LLama 2.0 Pre-requisites:
 - Download the dataset. The command below downloads the dataset to the path ```/home/ubuntu/examples_datasets/``` and further commands in this ReadMe assume that path for dataset. If the script is modified OR the data is placed in some other path, the commands need to be adjusted as appropriate.
 ```
 cd neuron-lightning-ray/data && python get_dataset.py --llama-version 2
 ```
+- Install the requirements
+```
+pip install -r requirements.txt
+Install neuronx-distributed: python -m pip install neuronx_distributed --extra-index-url https://pip.repos.neuron.amazonaws.com
```
Comment (PradeepKadubandi) [inline ReadMe.md:18]: This shouldn't be needed when we use the DLAMI right? If not, there are more installment steps required.

**Review by PradeepKadubandi — CHANGES_REQUESTED** [review]
Looks good to me mostly. Let's not add the steps for installing NxD manually (if we want to add them, we must add all the other setup needed when not using DLAMI as well)

File: ReadMe.md (hunk 1 above)
Comment (sindhupalakodety) [reply ReadMe.md:18]: i dont think with DLAMI neuronx-distributed is included and the requirements.txt has ray and correct transformer version in it, which are essential.

Comment (PradeepKadubandi) [reply ReadMe.md:18]: I don't remember installing `neuronx-distributed` but I could have forgotten. Let me approve the PR so that you can merge.

**Review by PradeepKadubandi — APPROVED** [review]
Let's merge.

**Pull request merged**